packages/cli/README.md

```diff
@@ -76,12 +86,19 @@ features](https://github.com/darvid/trine/wiki/YAML-Primer).
    	* `smartthings capabilities --help`,
    	* `smartthings capabilities:presentation --help`
    	* `smartthings capabilities:presentation:create --help`
-1. In general, the CLI accepts JSON data in YAML or JSON format and will
-   output data in either format as well.
+1. The CLI accepts data in YAML or JSON format and can
+   output data in either format as well as the default table format.
+1. Commands that take input accept stdin or a file specified
+   by the `--input` (shortcut `-i`) flag.
+1. Commands that output data will output the data to stdout unless a file
+   is specified the using `--output (shortcut `-o`) flag.
+1. When command takes input and results in output, the format of the output
```

Comment:
Either this should be 
> When a command. . .
or we make command plural and edit all subject-verb agreement. My only input is we should try to stay consistent and the above list items are plural with commands.